playwright-e2e/authentication/auth-singular.ts

```diff
@@ -15,7 +15,7 @@ export async function fillEmailPassword(
   const passwordInput = page.locator('input[type="password"]');
   await passwordInput.fill(password);
 
-  const navPromise = waitForNavigation ? page.waitForNavigation({ waitUntil: 'domcontentloaded' }) : Promise.resolve();
+  const navPromise = waitForNavigation ? page.waitForNavigation({ waitUntil: 'load' }) : Promise.resolve();
```

Comment:
Since I will update Playwright version soon, I'm going to make [this](https://playwright.dev/docs/release-notes#behavior-change) mentioned change now.

Reply:
Will this introduce any breakage in the tests until the Playwright version is updated?